README.md

```diff
@@ -220,4 +226,38 @@ Not all steps need to be followed on the instructions for the aad-pod-identity p
     secret1
     ```
 
+#### OPTION 3: VMSS User Assigned Managed Identity
+
+This option allows flexvol to use the user assigned managed identity on the k8s cluster VMSS directly.
+
+### Prerequisites:
+
+Make sure you have AKS with [managed identities](https://docs.microsoft.com/en-us/azure/aks/use-managed-identity) 
+
+1. Get the user identity that created automaticaly by AKS
```

Comment:
can we also add the step to create the managed identity if user not using an AKS cluster?

`az identity create -g <RESOURCE GROUP> -n <IDENTITY NAME>`

For AKS clusters with managed identity, the user can get the client id with -

`az aks show -g <resource group> -n <aks cluster name> --query identityProfile.kubeletidentity.clientId -o tsv`